utils/mentions.py

```diff
@@ -0,0 +1,46 @@
+import re
+from enum import Enum
+
+from discord.ext.commands import Converter
+
+from models import db_session
+from models.user import User
+
+__all__ = ["MentionConverter"]
+
+
+class MentionType(Enum):
+    ID = (0,)
```

Comment:
Why is this a tuple?

Reply:
Good question, I have no idea how that happened